docs/howto/index.md

```diff
@@ -7,7 +7,7 @@
 
 Install authd <install-authd>
 Configure authd <configure-authd>
-Login with GDM <login-gdm>
-Login with SSH <login-ssh>
+Log in with GDM <log-in-gdm>
+Log in with SSH <log-in-ssh>
```

Comment:
Ah you also need to rename the docs or this will fail, or keep the old doc name for the link only.

```suggestion
Log in with GDM <login-gdm>
Log in with SSH <login-ssh>
```

Reply:
Thanks @3v1n0 -- yes, this is causing a linkcheck build error.

@giuliazanchi -- I would suggest keeping the old doc name as Marco has suggested above -- this refers to the actual markdown filename in `/howto/`. The user won't see this anyway.

If you want to test this kind of thing locally use `make linkcheck` in the `/docs/` directory of the repo.